lang/en/docs/materials-designer/header-menu/edit.md

```diff
@@ -27,3 +27,11 @@ The user can also `Reset` <i class="zmdi zmdi-close zmdi-hc-border"></i> all cha
 The `Edit` menu also allows the user to `Clone` <i class="zmdi zmdi-collection-image zmdi-hc-border"></i> the currently selected structure into a new distinct entry in the items list sidebar. This feature is demonstrated in the short animation below, where we clone the original silicon structure into a new entry which appears below it:
 
 <img data-gifffer="/images/materials-designer/edit-clone.gif" />
+
+## Use Conventional Cell
+
+The `Edit` menu also allows the user to update the material ot use a `Conventional Cell`.
+
+## Toggle "isNonPeriodic"
+The 'Edit' menu also allows the user to update the material to be categorized as `non-periodic`. By clicking the `Toggle "isNonPeriodic"` button the materials lattice and basis will be updated so that it's coordinates are centered inside a cubic lattice with dimensionality based on the size of the structure. Upon saving the basis of the stucture will be set to Cartesian Units. the difference between a `periodic` and `non-periodic` structure is denoted by a change in the icon representing the structure. `Periodic` structures are denoted by <i class="zmdi zmdi-widget zmdi-hc-border"></i> icon while `non-periodic` structures are denoted by <i class="zmdi zmdi-device-hub zmdi-hc-border"></i> icons.
```

Comment:
I'd replace the last sentence starting with `Periodic` with a reference to https://github.com/Exabyte-io/documentation/pull/219/files#diff-287fca2a1961b4dc4531fdc461d08b6db12f024ad839edaedc49a8577ba6e9ffR28 instead to have one source of truth.